load/load_test.go

```diff
@@ -413,3 +413,13 @@ func TestLoadEmptyText(t *testing.T) {
 	)
 	require.NoError(t, err, "should return no error")
 }
+
+func TestLoadCSV_E2023(t *testing.T) {
+	err := Load(&unittestpb.ItemConf{}, "../testdata/", format.CSV,
+		SubdirRewrites(map[string]string{"unittest/Unittest": "unittest/Unittest2"}),
+	)
+	require.Error(t, err, "should return an error")
+	desc := xerrors.NewDesc(err)
+	require.Equal(t, "E2023", desc.ErrCode())
+	t.Logf("error: %s", desc.String())
+}
```

Comment:
Should add test case to `confgen` package (same package of the logic code), otherwise it will not contribute to test coverage.

> **Test coverage**: If your tests are in a different package, they will not contribute to the test coverage of the package being tested. This can lead to misleading coverage reports, as it may appear that certain parts of your code are not being tested.